resultsdbupdater/utils.py

```diff
@@ -360,14 +419,38 @@ def verify_topic_and_testcase_name(topic, testcase_name):
 
 
 def handle_ci_umb(msg):
+    #
+    # Handle messages in Fedora CI messages format
+    #
+    # https://pagure.io/fedora-ci/messages
+    #
+
     session = retry_session()
 
     msg_id = msg['headers']['message-id']
     msg_body = msg['body']['msg']
+
+    # check if required version is provided in the message
+    if 'version' not in msg_body:
```

Comment:
You may consider using default value "0.1.0" (and log a warning), otherwise it may break some nonconforming CI systems (our segment tester was unfortunately one of those).

You can list recently published CI messages with invalid version with following command.

```bash
http get https://apps.fedoraproject.org/datagrepper/raw delta==17280 rows_per_page==100 category==ci |
    jq '.raw_messages[] | select(.msg.version as $in | ["0.1.0", "0.2.0", "0.2.1"] | index($in) | not)'
```

Reply:
I would ask to require version field instead. Note that resultsdb updater is used only on downstream

Reply:
@lcarva what is it breaking? CVP?

Reply:
That's one of the things, yes.

Reply:
CVP itself is not breaking. It is the team's messages that are submitted whose results that CVP collates together is breaking. Seems our examples and docs omitted the version field and that was copied by said teams. I have updated the docs and reached out to the teams in question.